resources/themes/cura-light/theme.json

```diff
@@ -123,6 +128,11 @@
             "italic": true,
             "family": "Noto Sans"
         },
+        "tiny": {
+            "size": 0.85,
+            "weight": 40,
+            "family": "Noto Sans"
+        },
```

Comment:
Need to think twice about adding this new size. We've had great efforts to restrict the available font sizes to make the interface look consistent, and having many different font sizes is detrimental to that.

This is ultimately part of the design, but in an interest to keep the design consistent for the future, it would be good to reconsider this font size. Is "small" not good enough, or "default"? How is this going to look in a Japanese, Chinese or Taiwanese font?

This is something you'll need to discuss with the designer. Make sure that he knows beforehand which set of font sizes he wants to use and *then make him use it across the entire interface* instead of giving unfettered freedom to use whichever font size is the flavour of the day. This makes maintenance easier in the future and keeps the interface consistent. We currently have 5 font sizes: Small, default, medium, large and huge. Do we really want to add a 6th? I'd think that the default, which is 0.1em larger, should do fine.

(Not to mention that "tiny" is bigger than "small" apparently.)